Services/GEV/Desktop/classes/class.gevAgentOfferGUI.php

```diff
@@ -56,9 +56,8 @@ public function executeCommand() {
 			default:
 				throw new ilException("gevAgentOfferGUI: Can't forward to '$next_class'");
 		}
-		
-		$this->tpl->setContent($ret);
-		$this->tpl->show();
+		echo json_encode($ret);
```

Comment:
perhaps patch markers?

Reply:
nope. It's an gev File and not once of the core.